master/buildbot/test/unit/test_reporters_words.py

```diff
@@ -518,7 +518,8 @@ def test_command_watch_builder0_get_notifications(self):
         yield self.sendBuildFinishedMessage(16)
         self.assertEqual(len(self.sent), 1)
         self.assertIn(
-            'Hey! build builder1 #6 is complete: Success []', self.sent)
+                "Build builder1 #6 is complete: Success [] - "
```

Comment:
This comes from the master. in the IRC message it contains either red or green state_string (e.g. finished,abort,etc) if colorization is enabled. If not, the status code is plain text. see https://github.com/buildbot/buildbot/blob/master/master/buildbot/reporters/words.py#L559

In the test case the status seems to return an uncolored empty string.